tests/data_template/module_080/i18n/fr.po

```diff
@@ -0,0 +1,24 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+#	* odoo_module_migrator
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 8.0+e\n"
```

Comment:
Side question, what is +e ?

Reply:
enterprise ?

Reply:
> Side question, what is +e ?

Good question ! 
I have no idea.

I used the export translation wizard from Odoo and it seems that the +e is there, even when not enterprise :
![image](https://user-images.githubusercontent.com/32102436/126123369-560af399-1fc8-45cd-8339-d6498147139e.png)

Running through a few other po files, it is not always there though.

Reply:
> enterprise ?

You are right @yelizariev 

This comes from the fact that web_enterprise is installed on the database I used to export my translation.

https://github.com/odoo/odoo/blob/28c3f51c4878fbcd79b2e819948465fcf2160ebc/odoo/tools/translate.py#L588

https://github.com/odoo/enterprise/blob/a27e3599eaca2c12344e68c034de5ae20a03ff4c/web_enterprise/version.py#L14

Reply:
very nitpicking : remove both ``+e`` as it has no sense for 8.0.

Reply:
I don't think this change should be done, as it doesn't impact on the use of the translations, and it will generate a big diff on the migration commit. On pre-commit commit, it can be tolerated though.

Reply:
Hi pedro. I'm not sure to understand. do you talk about the "+e", or regarding the general idea of the PR ? 
Anyway, I don't understand the "big diff" : 
- It's just one line, in each translated file.
- I guess that it will done later anyway by OCA translate tools.

Reply:
The general PR. It's not done by ocabot, as that header information is not needed for anything. When you apply the new POT over all .po, that's not changed. It's a big diff regarding number of changed files when you review the migration commit and you fin 80 files changed, while 60 of them belongs to .po. It's to make a lot more dirty something that only serves for human read.

Reply:
@pedrobaeza : makes sense. (even if modules with 80 translation files are not in the majority (my feeling)) :+1: 

@baimont  : could you explain why you made this PR ? it's just for "cosmetic" reason, or the id of the version is important for anything ?

Reply:
I agree with Pedro. It doesn't look important  patch translation files.

Anyway, I think the script should be in a separate file, e.g. `bump_version_i18n.py`. This way it would be easier to disable it.

Reply:
> @baimont : could you explain why you made this PR ? it's just for "cosmetic" reason, or the id of the version is important for anything ? I just rgreped odoo and OCA modules, and `Project-Id-Version` seems useless.

@legalsylvain I am doing a migration from 10.0 to 14.0 at the moment and this is a change I have to make manually for each module I migrate. It was just a way for me to learn a little bit more about the tool I am using and make that particular step automatic at the same time. If it's controversial, I guess it'd better stay a customization for me :-D

Reply:
@baimont but that's the question: you don't have to do such operation. It's only something aesthetic, but not required for the normal behavior and not affecting translations per se.